### ISSUE TITLE
bndcmpr playlists?

### ISSUE THREAD

**Issue by natumbri** [impliedchaos/mopidy-bandcamp#22]
Any interest in supporting bndcmpr playlists?

see https://github.com/natumbri/mopidy-bandcamp/commit/fb1dd93df200d5d1a679c5ae62c9a6e8f89dd189

Cheers,
Nik

**Comment by impliedchaos** [comment]
Absolutely!   I'm overloaded with work right now, but I'll try to get this merged and the docs updated this weekend.  Thanks!

**Comment by natumbri** [comment]
If you're interested, I can put together a proper pull request that is a bit tidier and limited to strictly what is required. 

One question: how do you feel about multi-threading? Without multi-threading, it takes a while to look up a long playlist; parallelizing makes it quicker, obviously, but involves adding a new import, and hitting bandcamp.com harder.

There is probably an even better alternative, to process the bndcmpr playlist sequentially using pykka futures, so that the mopidy queue is updated as each track is processed, instead of all at once. But that is beyond my capacity right now.

**Comment by impliedchaos** [comment]
Hi Nik,

Yeah, a PR would be fantastic.

And, I don't have a problem with threading.  

Thanks,
Dave
